### ISSUE TITLE
Feat Color Contrast Ratio

### ISSUE THREAD

**Issue by rodrigorhas** [Toinane/colorpicker#37]
Option to show color Contrast ratio like chrome color picker

**Comment by Toinane** [comment]
Yeah I’ve already thinking to implement this type of feature. I need to find how I can add this type of feature into colorpicker
